controls/radfiledialogs/features/network-locations.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Network Locations Support
+page_title: Network Locations Support
+description: Network Locations Support
+slug: radfiledialogs-features-network-locations
+tags: network,locations
+published: True
+position: 5
+---
+
+# Network Locations
+
+Since **R3 2018**, the RadFileDialogs as well as the ExplorerControl provide out of the box support for Network Locations.
+
+## Breaking Changes
```

Comment:
Can be place this at the bottom of the article? The article must show how the feature works. The breaking changes are something that is aside of the main idea of the article.